docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -62,6 +63,58 @@ to return the metadata pertaining to a partition or column respectively.
     
     **Syntax:** `[ database_name. ] [ table_name. ] column_name`
 
+* **AS JSON**
+
+  An optional parameter to return the table metadata in JSON format. Only supported when EXTENDED
+  format is specified.
+
+  **Syntax:** `[ AS JSON ]`
+
+  **Schema:**
+  
+  ```sql
+  {
+    "table_name": "<table_name>",
+    "catalog_name": [...],
+    "database_name": [...],
+    "qualified_name": "<qualified_name>"
+    "type": "<table_type>",
+    "provider": "<provider>",
+    "columns": [
+      {
+      "id": 1,
+      "name": "<name>",
+      "type": <type_json>,
+      "comment": "<comment>",
+      "default": "<default_val>"
+      }
+    ],
+    "partition_values": {
+      "<col_name>": "<val>"
+    },
+    "location": "<path>",
+    "view_definition": "<view_defn>",
```

Comment:
if it's not a view, will this field still be present with value as empty string?